my_robot_model:

  joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 20

  wheel_LF_joint_velocity_controller:
      type: velocity_controllers/JointVelocityController
      joint: wheel_LF_joint

  wheel_RF_joint_velocity_controller:
      type: velocity_controllers/JointVelocityController
      joint: wheel_RF_joint

  # wheel_LB_joint_velocity_controller:
  #     type: velocity_controllers/JointVelocityController
  #     joint: wheel_LB_joint
  #
  # wheel_RB_joint_velocity_controller:
  #     type: velocity_controllers/JointVelocityController
  #     joint: wheel_RB_joint
  #
  front_axle_joint_position_controller:
      type: velocity_controllers/JointPositionController
      joint: front_axle_joint
      pid: {p: 1000.0, i: 1.0, d: 0.0}

  gazebo_ros_control:
      pid_gains:
        wheel_LF_joint: {p: 5.0, i: 1.0, d: 0.0}
        wheel_RF_joint: {p: 5.0, i: 1.0, d: 0.0}
        # wheel_LB_joint: {p: 5.0, i: 1.0, d: 0.0}
        # wheel_RB_joint: {p: 5.0, i: 1.0, d: 0.0}
        front_axle_joint: {p: 1000.0, i: 1.0, d: 0.0}
